modules/wrapper/rc/config.nix

```diff
@@ -133,6 +133,21 @@ in {
       configRC = {
         globalsScript = entryAnywhere (concatStringsSep "\n" globalsScript);
 
+        # Call additional lua files with :luafile in Vimscript
+        # section of the configuration, only after
+        # the luaScript section  has been evaluated
+        extraLuaFiles = let
+          callLuaFiles =
+            map (
+              file:
+                if builtins.isPath file
```

Comment:
`builtins.isPath` doesn't check if a file exists, it only checks if the type is `path`. 
If it is a path, it must be valid otherwise it will cause an evaluation error before it reaches this code.
Paths that are `str` will always throw for the same reason above.